src/main/scala/com/microsoft/partnercatalyst/fortis/spark/ProjectFortis.scala

```diff
@@ -0,0 +1,194 @@
+package com.microsoft.partnercatalyst.fortis.spark
+
+import com.microsoft.partnercatalyst.fortis.spark.logging.AppInsights
+import com.microsoft.partnercatalyst.fortis.spark.pipeline._
+import com.microsoft.partnercatalyst.fortis.spark.streamprovider.ConnectorConfig
+import com.microsoft.partnercatalyst.fortis.spark.transforms.image.{ImageAnalysisAuth, ImageAnalyzer}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.language.{LanguageDetector, LanguageDetectorAuth}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.locations.{Geofence, LocationsExtractor, PlaceRecognizer}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.locations.client.FeatureServiceClient
+import com.microsoft.partnercatalyst.fortis.spark.transforms.sentiment.{SentimentDetector, SentimentDetectorAuth}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.topic.KeywordExtractor
+import org.apache.log4j.{Level, LogManager, Logger}
+import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+import sun.reflect.generics.reflectiveObjects.NotImplementedException
+
+object ProjectFortis extends App {
+
+  private val pipelines = List[Pipeline](
+    InstagramPipeline,
+    RadioPipeline,
+    TwitterPipeline,
+    FacebookPipeline,
+    TadawebPipeline
+  )
+
+  // TODO: create helper method to augment Settings object with data from Cassandra
+  // TODO: move most of these settings to Cassandra
+  private implicit object Settings extends Settings {
+    import scala.util.Properties.envOrNone
+
+    val progressDir: String = envOrFail(Constants.Env.HighlyAvailableProgressDir)
+    val featureServiceHost: String = envOrFail(Constants.Env.FeatureServiceHost)
+    val oxfordLanguageToken: String = envOrFail(Constants.Env.OxfordLanguageToken)
+    val oxfordVisionToken: String = envOrFail(Constants.Env.OxfordVisionToken)
+
+    val appInsightsKey: Option[String] = envOrNone(Constants.Env.AppInsightsKey)
```

Comment:
Shouldn't the compiler be able to infer these types? Why do we need to declare them? The type is pretty obvious from the name so declaring it doesn't add much value.